content/index.md

```diff
@@ -33,8 +34,11 @@ that are [reusable across multiple languages and frameworks](#portability).
 
 </div>
 </div>
+<figcaption>
+    This <code>Card</code> component is <a href="#components">implemented as a JavaScript function</a>.
```

Comment:
I've learned from you that we could use Markdown here:

```suggestion
<figcaption>

This `Card` component is [implemented as a JavaScript function](#components).

</figcaption>
```

Reply:
:) The reason this works is that the Markdown gets wrapped in a `<p>` Tag which is then inserted into the markup at that point.

Reply:
Oh, good point: I hadn't considered the additional `<p>` tag.

Just to clarify: So you're saying that's not desirable though? (I tend to agree; it seems inappropriate.)

FWIW, I'm still not sure why commonmark permits this in the first place; might be unintentional. 🤷‍♂

Reply:
I'm on the fence about the tradeoffs of an extra `<p>` tag here. In the case of the jsx code blocks, you actually get a `<pre>` block, which I think is very appropriate.

The positive about the `<p>` tag is that you can use markdown to write your text. However in this instance, because the text is within the `<figcaption>` and I am styling `<p>` Tags within my `.header-area` (giving them slightly bigger text, since they should be more noticable), I might have to do style resetting on `<p>` Tags within my `<figcaption>`. Style resetting is always a code smell for me (means my scoping is probably off), but as I said, it's a tradeoff in this case.

Reply:
I'd say it's not worth it: Proper markup overrides such minor convenience enhancements here. In the imaginary Gorgeon future, we'd solve this with proper component parameters anyway, in which case we certainly wouldn't choose to insert an additional `<p>` for this (i.e. I agree it's a smell).